index.html

```diff
@@ -1,14 +1,113 @@
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+
+  <title>medhub</title>
 </head>
 <body>
-
+<div class="container">
 	<img src="https://www.medhub.com/wp-content/uploads/2016/10/Medhub_Logo_CMYK.png" width="200px" height="auto">
-	<h1 class="header">My medHub</h1>
+	<h1 class="header">VIAGRA</h1>
+</div>
+	<div class="info">
+		<div> 
+			<p>Despre Viagra</p>
+		</div>
+		<div class="vl"></div>
+		<div>
+			<p>Contacteaza-ne</p>
+		</div>
+		<div class="vl"></div>
+		<div class="romania">
+			<p> ROMANIA </p>
+		</div>
+		<div class="vl"></div>
+		<div class="search-container">
+    <form action="/action_page.php">
+      <input type="text" placeholder="Cautare.." name="search">
+      <button type="submit"><i class="fa fa-search"></i></button>
+    </form>
+  </div>
+		
+	</div>
 	
-	<p>My first paragraph.</p>
+	<div class="main-container">
+		<div class="image">
+			<img src="https://ubmidrupalcandd.s3.amazonaws.com/styles/article_image_620x420/s3/public/CD%20Image%20Library/Editorial/Clinical/Products%20and%20packshots/Viagra-Connect_620.jpg?itok=10aNPex5">
+			
+		</div>
+		<div class="info-container">
+			<h2 class="container-title">Viagra 50 mg</h2>
+			 <div class="panel-group" id="accordion">
+    <div class="panel panel-default">
+      <div class="panel-heading">
+      	 <h4 class="panel-title">
+          <a data-toggle="collapse" data-parent="#accordion" href="#collapse1">Descriere</a>
+        </h4>
+      </div>
+      <div id="collapse1" class="panel-collapse collapse in">
+        <div class="panel-body">Lorem ipsum dolor sit amet, consectetur adipisicing elit,
+        sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+        quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</div>
+      </div>
+    </div>
+     <div class="panel panel-default">
+      <div class="panel-heading">
+        <h4 class="panel-title">
+          <a data-toggle="collapse" data-parent="#accordion" href="#collapse2">Ingrediente</a>
+        </h4>
+      </div>
+      <div id="collapse2" class="panel-collapse collapse">
+        <div class="panel-body">Lorem ipsum dolor sit amet, consectetur adipisicing elit,
+        sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+        quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</div>
+      </div>
+    </div>
+				
+	 <div class="panel panel-default">
```

Comment:
I think panel-default, is used for a default behavior. Which means when you have 4 elements, you have to set a default value, usually it's the first element, so in your case the default value should be the Descriere, which has the default value :)